superset/assets/src/dashboard/stylesheets/dashboard.less

```diff
@@ -129,10 +129,25 @@ body {
   }
 }
 
-.modal img.loading {
-  width: 50px;
-  margin: 0;
-  position: relative;
+.modal {
+  img.loading {
+    width: 50px;
+    margin: 0;
+    position: relative;
+  }
+
+  .delete-modal-actions-container {
+    margin: 40px 0 12px 0;
+
+    .btn {
+      margin-right: 24px;
```

Comment:
could we cut this in half?

Reply:
i fixed padding, message text, and a few modal CSS. please see the updated attachment.